routes/error-tracker.js

```diff
@@ -40,19 +40,57 @@ const GAE_METADATA = {
   severity: 500, // Error.
 };
 
+/** Extract parameters from a GET or POST request. */
+function getRequestParams(req) {
+  return req.method === 'POST' ? req.body : req.query;
+}
+
 /** Logs an event to Stackdriver. */
-function logEvent(log, event) {
-  return new Promise((resolve, reject) => {
-    log.write(log.entry(GAE_METADATA, event), writeErr => {
-      if (writeErr) {
-        reject(writeErr);
-      } else {
-        resolve();
-      }
-    });
-  });
+async function logEvent(log, event) {
+  await log.write(log.entry(GAE_METADATA, event));
```

Comment:
Does `log.write` return a promise now?

Reply:
Yes it does (although I'm not sure when that was introduced, so I updated the logging library, along with a bunch of other very outdated deps). [The docs] however suggest not awaiting at all because the library will batch writes to improve throughput, so I'm updating this and the try/catch code to use `then`/`catch` (if necessary for debug) instead of `await`